managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/MetaStoreImpl.java

```diff
@@ -316,10 +317,11 @@ private static MetaStoreException getException(Throwable t) {
             metadataByteBuf.writeShort(MAGIC_MANAGED_LEDGER_INFO_METADATA);
             metadataByteBuf.writeInt(mlInfoMetadata.getSerializedSize());
             metadataByteBuf.writeBytes(mlInfoMetadata.toByteArray());
-
+            byte[] byteArray = managedLedgerInfo.toByteArray();
+            uncompressedByteBuf = Unpooled.directBuffer(byteArray.length);
+            uncompressedByteBuf.writeBytes(byteArray);
```

Comment:
Why we need a copy here？

Reply:
Please add some comments to avoid being optimized for wrappers later by other developers

Reply:
`Unpooled.wrappedBuffer` will create ByteBuf on heap and the `hasMemoryAddress` will return `false`. 
`Unpooled.directBuffer` will create ByteBuf on `PooledUnsafeDirectByteBuf`, and the `hasMemoryAddress` will return `false`.